pyaerocom/aeroval/coldatatojson_helpers.py

```diff
@@ -821,14 +820,13 @@ def _process_map_and_scat(
     use_fairmode,
     obs_var,
 ):
-
     stats_dummy = _init_stats_dummy()
     scat_data = {}
     scat_dummy = [np.nan]
     for freq, cd in data.items():
-        use_dummy = True if cd is None else False
         for per in periods:
             for season in seasons:
+                use_dummy = cd is None
```

Comment:
if `use_dummy = cd is None` is confusing maybe it would be more readable to replace
`if not use_dummy` with `if cd is not None`.

Reply:
It's fine. This is me using several languages at once and therefore trying not to get too much into language specifics. After 2 seconds I got it.
I had just moved the existing line without much thinking about it.